torch/testing/_internal/common_methods_invocations.py

```diff
@@ -234,6 +235,7 @@ def __init__(self,
             self.supports_complex_autograd = supports_autograd
 
         self.gradcheck_wrapper = gradcheck_wrapper
+        self.check_gradgrad = check_gradgrad
```

Comment:
Would "supports_gradgrad" be more accurate?

If so, what about modeling the default value as None and inheriting from `supports_autograd` by default? Currently if someone set `supports_autograd=False`, for example, the `check_gradgrad` would still be `True`.

Reply:
I wasn't sure how to name it. I used this to match the existing `check_batch_grad` and `check_batch_gradgrad` naming convention.

All 3 of these args are only checking if `supports_autograd=True` and ignored otherwise.

Reply:
OK; let's just go with "supports_gradgrad", which is a statement about the operator (whereas check_batch_grad is a testing directive) and just extend the comment to note that the value is ignored if supports_autograd is false

cool?

Reply:
Sounds good!